pkg/image/registry/imagestreamtag/rest.go

```diff
@@ -333,11 +333,14 @@ func newISTag(tag string, imageStream *imageapi.ImageStream, image *imageapi.Ima
 			Name:              istagName,
 			CreationTimestamp: event.Created,
 			Annotations:       map[string]string{},
+			Labels:            imageStream.Labels,
```

Comment:
I think we need to decide to return labels, so that you can select image stream tags by labels.

Not supporting labels is "not-kube".  Agree there is some things we could do - but I don't think that not including them is a good strategy.